docs/framework/get-started/system-requirements.md

```diff
@@ -24,7 +24,7 @@ manager: "wpickett"
 # .NET Framework System Requirements
 The tables in this topic provide the hardware, operating system, and software requirements for the .NET Framework 4.5 and its point releases (4.5.1 and 4.5.2), the [!INCLUDE[net_v46](../../../includes/net-v46-md.md)] and its point releases (4.6.1 and 4.6.2), and the .NET Framework 4.7. Development environments that enable you to develop applications for the .NET Framework have a separate set of requirements.  
   
- For download information and links, see [Installation Guide](../../../docs/framework/install/guide-for-developers.md).  
+ For download information and links, see [Install the .NET Framework for developers](../../../docs/framework/install/guide-for-developers.md).  
```

Comment:
again, change link to index.md instead of changing link text. review where you made this change to see whether it makes sense to go to the developer's one or the general one